### PR TITLE
[MIRROR] Fixes weakref runtimes

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/34195
--------------------

### CODE REVIEW

**Comment by Pyko1** [comment]
Lets see if it works now.

Also there was some hippieshit in there that was not marked, or at least the TG file diditn have em.
Someone else codecheck, also i have no idea if that hippie shit is needed or not, i just fixed the merge conflicts in file.

**Comment by Carbonhell** [comment]
I think only update_teeth is needed, i don't understand why that grant_language line was added...huh.

**Comment by Pyko1** [comment]
maybe remove 
`grant_language(/datum/language/common) // ME TARZAN, YOU JANEBOT`
or at least i don't recall any specific reason why we would need it.

**Comment by Pyko1** [comment]
https://github.com/tgstation/tgstation/pull/26776
TG removes the line.

**Comment by Pyko1** [comment]
https://github.com/tgstation/tgstation/commit/7554b5f4541231ddbdc77e66a493fc04752c93e4#diff-1ff8328c73a4a8a73ad2c1fe6d517588
TG ads the line

**Comment by Pyko1** [comment]
pretty sure its from some poor handling of merge conflicts.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -25,11 +25,14 @@
 	create_internal_organs() //most of it is done in set_species now, this is only for parent call
 
 	handcrafting = new()
-	AddComponent(/datum/component/redirect, list(COMSIG_COMPONENT_CLEAN_ACT), CALLBACK(src, .proc/clean_blood))
-
+	//hippie start - <unknown, teeth apparently?>
 	grant_language(/datum/language/common) // ME TARZAN, YOU JANEBOT
```
Comment (JohnGinnane) [inline code/modules/mob/living/carbon/human/human.dm]: why do we still have this?

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**